Unity-Technologies/com.unity.multiplayer.samples.bitesize
# [MTT-8894] Adding Main Menu UI for Distributed Authority Sample

### Description


This PR adds a UI Design for the Main Menu scene. The original Chop Chop main menu scene got imported and modified (colliders removed, some positions changed and some assets removed). Also the UITK sample main menu UI got imported and modified. 
Functionality for Servicehelper.cs and MainMenuView.cs got adjusted to handle connection logic.

How to test:
Open Bootstrap scene and press play (after connection Vivox in project settings to a valid project id).
Check that entering Player name, Session name and pressing Start button in Main Menu work as expected.

Considerations:
We might want to monitor which kind of player names people can enter (see: https://unity.slack.com/archives/C023C5V8P9S/p1726586223121339)


### Issue Number(s)

https://jira.unity3d.com/browse/MTT-8894

### Contribution checklist
